### PR TITLE
[DONOT-MERGE] :Remove build number from package names . (release/2.0.0)

### PR DESCRIPTION
**Pull request by rakeshsinghranchi**
Generate installers that don't have the build number in file name. This change also takes care for nugget packages. 
Test is in progress and PR is intentional to get the feedbacks on the items to change. 
Based on the feedback, I will make the build number suffix to be configurable via a switch.

### CODE REVIEW

**Comment by rakeshsinghranchi** [comment]
Successfully tested Windows installer and nuget packages.

**Comment by weshaggard** [comment]
We should hold on merging this PR until we have final preview1 packages from corefx and coreclr and include the version updates as part of this PR (or before this PR).

**Comment by rakeshsinghranchi** [comment]
@gkhanna79 would you like to make the necessary change here to reduce the turnaround time? Please suggest.

**Comment by weshaggard** [comment]
>I think this is fine for a targeted change for tomorrow given that master (which will be preview2) has already undergone a change.

@gkhanna79 do you want to pull in the corefx/coreclr package update into this PR? If not then we at least need to wait until that is merged in before merging this PR.

**Comment by weshaggard** [comment]
We also need to make sure to get CI green.

**Comment by gkhanna79** [comment]
Once CoreCLR/CoreFX packages are published, we will update this PR with them to ensure CI is green and then merge this.

**Comment by rakeshsinghranchi** [comment]
@gkhanna79  can you take a look at the CI failures. As part of testing, I could successfully build Windows installer with the build number removed. What else do we need to test ?

**Comment by Petermarcu** [comment]
@rakeshsinghranchi, @ramarag it looks like the tests are expecting '2.0.0-preview1-002104-00'.

**Comment by ramarag** [comment]
the build is  producing 2.0.0-preview1.nupkg, and nuget packages from the feed are resolved as higher versions. i beleive we need to pass the correct build numbers in the build

**Comment by Petermarcu** [comment]
This is the test build for where the shared framework will not have a build number. It will just be 2.0.0-preview1.

File: TestAssets/TestProjects/LightupClient/project.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
       "dependencies": {
         "Microsoft.NETCore.App": {
           "type": "platform",
-          "version": "2.0.0-preview1-*"
+          "version": "2.0.0-preview1*"
```
Comment (gkhanna79) [inline TestAssets/TestProjects/LightupClient/project.json]: This (and others) should be "2.0.0-preview1", without any asterisks.

**Comment by gkhanna79** [comment]
@rakeshsinghranchi have you validated that the nuget packages produced have correct dependency on each other (e.g. M.N.App 2.0.0-Preview1 depends upon M.DotnetHost.Hostpolicy 2.0.0 Preview1 that depends upon same version of HostFXR, which depends upon AppHost)?

Also, you will need to fix the version of DependencyModel and PlatformAbstractions nuget packages under src folder.

**Comment by gkhanna79** [comment]
Please fix versioning of https://github.com/dotnet/core-setup/tree/release/2.0.0/src/Microsoft.DotNet.PlatformAbstractions and https://github.com/dotnet/core-setup/tree/release/2.0.0/src/Microsoft.Extensions.DependencyModel to be preview1 as well.

**Comment by gkhanna79** [comment]
@dotnet-bot test Fedora24 x64 Debug Build please

**Comment by rakeshsinghranchi** [comment]
@gkhanna79 updated the PR with the feedback comments. Also, @ramarag helped look at the nuget package dependencies. Note that I haven't kicked-off the pipeline test build for this change. Do we have a checklist to verify for this branding change?

**Comment by weshaggard** [comment]
decided not to do this.

**Pull request closed without merging**